driver/node.go

```diff
@@ -236,7 +236,7 @@ func (s *nodeService) createDeviceIfNeeded(device string, lv *proto.LogicalVolum
 	switch err {
 	case nil:
 		// a block device already exists, check its attributes
-		if stat.Rdev == unix.Mkdev(lv.DevMajor, lv.DevMinor) && (stat.Mode&devicePermission) == devicePermission {
+		if stat.Rdev == int32(unix.Mkdev(lv.DevMajor, lv.DevMinor)) && (stat.Mode&devicePermission) == devicePermission {
```

Comment:
For 365, amd64, arm, and arm64, Stat_t.Rdev is int64.
What is your build arch (GOARCH)?